db/data.js

```diff
@@ -1347,7 +1340,7 @@ const produce = [
     imageURL:
       "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637897/Seasonal/052-strawberry_pxugsj.png",
     description:
-      "A popular sweet, juicy seasonal berry enjoyed in cuisines around the world. But Botanically, the strawberry fruit is considered an “accessory fruit” and is not a true berry. Strawberries are actually a group of fruits (genus Fragaria) containing several varieties and are part of the rose family. Strawberries are usually sweet with a firm and juicy but not tough flesh. The tiny seeds add a light crunch. Flavour, colour, size, and texture can vary depending on the variety of strawberry. They are an excellent source of vitamin C and manganese and also contain decent amounts of folate (vitamin B9) and potassium. The now-common garden strawberry didn't hit its stride until the 18th century, but local wild berries have been eaten since ancient Roman times. Modern strawberries are grown and eaten all over the world and can be served raw, cooked, or dried. Strawberries like fertile, well-drained, slightly acidic soil, and do best in full sun, but will tolerate some shade. Avoid planting in sites prone to frost, which can damage the flowers, or in exposed locations, which make it hard for pollinating insects to reach the flowers.",
+      "A popular sweet, juicy seasonal berry enjoyed in cuisines around the world. But Botanically, the strawberry fruit is considered an “accessory fruit” and is not a true berry. Strawberries are actually a group of fruits (genus Fragaria) containing several varieties and are part of the rose family. Strawberries are usually sweet with a firm and juicy but not tough flesh. The tiny seeds add a light crunch. Flavour, colour, size, and texture can vary depending on the variety of strawberry. They are an excellent source of vitamin C and manganese and also contain decent amounts of folate (vitamin B9) and potassium. The now-common garden strawberry did not hit its stride until the 18th century, but local wild berries have been eaten since ancient Roman times. Modern strawberries are grown and eaten all over the world and can be served raw, cooked, or dried. Strawberries like fertile, well-drained, slightly acidic soil, and do best in full sun, but will tolerate some shade. Avoid planting in sites prone to frost, which can damage the flowers, or in exposed locations, which make it hard for pollinating insects to reach the flowers.",
```

Comment:
Should we remove the double quotes around accessory fruit like we have done in other descriptions?

Reply:
Have done so, yes!